config/owasp/suppressions.xml

```diff
@@ -19,41 +19,9 @@
   <!--Please add all the temporary suppression under the below section-->
   <suppress>
     <notes>Temporary Suppression</notes>
```

Comment:
@kiran-yenigala-hmcts We can add the jira link in the notes for the respective CVEs. This will help us to map the CVE progress.